qpc/__init__.py

```diff
@@ -1,11 +1,20 @@
 """QPC Package Initialization."""
+import os
 from importlib import metadata
 from pathlib import Path
 from tomllib import load
 
 try:
-    # Let's get the package version from poetry
-    __package__version__ = metadata.version(__package__)
+    # Try to get the package version from importlib.
+    # This operation can pull its information from different places depending on how
+    # the program is being called. We cannot simply use `__package__` because the
+    # package might have been given a different name at install time, which we expect
+    # to match the QPC_VAR_PROGRAM_NAME value.
```

Comment:
this is odd. IMO we should be changing only the script name, not the whole package name.

Reply:
I'm trying a different change locally that may simplify the current hack workaround fix.

Reply:
I yanked this change after confirming locally that we can keep `qpc` as the hard-coded package name and expect `dsc` as the program entry point if `QPC_VAR_PROGRAM_NAME=dsc` was set at build/install time.